ext/AffineMapsChangesOfVariablesExt.jl

```diff
@@ -7,16 +7,16 @@ using AffineMaps
 using LinearAlgebra
 
 ChangesOfVariables.with_logabsdet_jacobian(f::Mul, x) = f(x), _mul_ladj(f.A, x)
-ChangesOfVariables.with_logabsdet_jacobian(f::InvMul, x) = f(x), - _mul_ladj(f.A, x)
+ChangesOfVariables.with_logabsdet_jacobian(f::InvMul, x) = f(x), .- _mul_ladj(f.A, x)
 
 ChangesOfVariables.with_logabsdet_jacobian(f::Add, x) = f(x), _add_ladj(x)
 ChangesOfVariables.with_logabsdet_jacobian(f::Subtract, x) = f(x), _add_ladj(x)
 
 ChangesOfVariables.with_logabsdet_jacobian(f::MulAdd, x) = f(x),  _mul_ladj(f.A, x)
-ChangesOfVariables.with_logabsdet_jacobian(f::InvMulAdd, x) = f(x), - _mul_ladj(f.A, x)
+ChangesOfVariables.with_logabsdet_jacobian(f::InvMulAdd, x) = f(x), .- _mul_ladj(f.A, x)
 
 ChangesOfVariables.with_logabsdet_jacobian(f::AddMul, x) = f(x),  _mul_ladj(f.A, x)
-ChangesOfVariables.with_logabsdet_jacobian(f::InvAddMul, x) = f(x), -  _mul_ladj(f.A, x)
+ChangesOfVariables.with_logabsdet_jacobian(f::InvAddMul, x) = f(x), .- _mul_ladj(f.A, x)
```

Comment:
Why the `.-` here?

Reply:
Ping @Micki-D

Reply:
Oh, I thought you had to broadcast that to negate the entries, but it works without the `.`
I'll change this

Reply:
Thanks - let  me know when it's ready to merge.